src/components/MultiSelectGroup/MultiSelectGroup.tsx

```diff
@@ -87,7 +82,10 @@ export const MultiSelectGroup: ChakraComponent<
             },
           }}
         >
-          {renderMultiSelect({ isBlockElement, multiSelectWidth: finalWidth })}
+          {renderMultiSelect({
+            isBlockElement: layout === "column",
+            multiSelectWidth,
```

Comment:
We can simply pass through `multiSelectWidth` here because (as stated in the comment above) if `fitContent` is passed, `MultiSelect` knows to render at a width of "100%" if the viewport is mobile. This means we probably do not need to have it be a prop on `MultiSelectGroup`, but I am not changing that in this PR.